scripts/seed_private_archive_from_prod.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+LOCAL_PRIVATE_ARCHVIE_DIR='/share/appdata/qgreenland-private-archive'
+PROD_PRIVATE_ARCHIVE_DIR='/share/appdata/qgreenland-private-archive/'
```

Comment:
Why this one have a trailing slash and before one not have?

Reply:
If the trailing slash is a requirement of the `rsync` command, I suggest moving the trailing slash to the command so the variables are consistent. The trailing slash right now is a hidden requirement that someone else might see as an error.

Reply:
Actually, this was just a mistake. I'll add the trailing slash to the first.